dockerfiles/app/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASE_IMAGE
+ARG BASE_IMAGE=ghcr.io/exabyte-io/centos-7.9-base
```

Comment:
Should we try `https://hub.docker.com/_/node`? 14-alpine

Reply:
Btw, I remeber that thish centos image comes from this Dockerfile: https://github.com/Exabyte-io/materials-designer/blob/dev/dockerfiles/centos/Dockerfile